spec/models/comment_spec.rb

```diff
@@ -292,6 +292,15 @@ def queue_adapter_for_test
         it "has comment_type \"fanwork-comment\"" do
           expect(subject.akismet_attributes[:comment_type]).to eq("fanwork-comment")
         end
+
+        it "has comment_post_modified_gmt as the work's revision time", :frozen do
+          work_posted_at = Time.current
+          travel_to(1.day.from_now) do
+            Chapter.new work_id: subject.commentable.id
+            expect(subject.akismet_attributes[:comment_post_modified_gmt]).to eq(Time.current.iso8601)
+            expect(subject.akismet_attributes[:comment_post_modified_gmt]).not_to eq(work_posted_at.iso8601)
```

Comment:
I think this secondary check is redundant since it will only fail here if time travel fails.